src/inference/src/dev/core_impl_ie.cpp

```diff
@@ -160,6 +98,8 @@ InferenceEngine::SoExecutableNetworkInternal ov::CoreImpl::LoadNetwork(
     const std::map<std::string, std::string>& config,
     const std::function<void(const InferenceEngine::CNNNetwork&)>& val) {
     OV_ITT_SCOPE(FIRST_INFERENCE, ie::itt::domains::IE_LT, "Core::LoadNetwork::Path");
+
+    // Need to have complex logic in order to differentiate v7 and v11 behavior
```

Comment:
I suppose in deprecated tests we don't use LoadNetwork with filename. 
Deprecated tests were deprecated before we introduced cache. So, there are no tests with IR v7 which use model cache. 
So, we can always redirect to `ov::ICore::compile_model`

Reply:
why do we have to return complex logic?

Reply:
I reverted these logic only in order to debug and fix tests